test/metric_value_benchmark/agent_configs/jmx_tomcat_jvm_config.json

```diff
@@ -4,7 +4,7 @@
   },
   "metrics": {
     "namespace": "MetricValueBenchmarkJMXTest",
-    "force_flush_interval": 5,
+    "force_flush_interval": 60,
```

Comment:
We should try updating the `metrics_collection_interval` to collect it more frequently instead of relying on the `force_flush_interval` https://github.com/aws/amazon-cloudwatch-agent-test/blob/main/test/metric_value_benchmark/agent_configs/jmx_kafka_config.json#L4

Reply:
addressed